style2.css

```diff
@@ -0,0 +1,404 @@
+ @media screen and (max-width: 1680px) {
+  body {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+@media screen and (max-width: 1440px) {
+  .main__pic {
+    margin: -100px 0 0 0;
+  }
+}
+
+@media screen and (max-width: 1120px) {
+  .header__menu-link {
+    font-size: 14px;
+  }
+
+  .main__pic {
+    margin: -80px 0 0 0;
+  }
+}
+
+@media screen and (max-width: 1090px) {
+  .header__menu-link {
+    font-size: 13px;
+  }
+
+  .main__pic {
+    margin: -70px 0 0 0;
+  }
+}
+
+@media screen  and (max-width: 1062px) {
+  .menu {
+    display: none;
+  }
+
+  .header__menu-link {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .body {
+    text-align: center;
+    align-items: center;
+  }
+
+  .main__pic {
+    margin: -50px 0 30px 0;
+  }
+	
+  .header {
+    display: flex;
+    padding: 0;
+    margin: 0 auto;
+  }
+  
+  .header__aside {
+    margin: 25px;
+  }
+
+  .main {
+    padding: 0;
+    margin: 0 auto;
+  }
+  
+  .header__menu {
+    display: none;
+  }
+
+  .header__menu-link {
+    display: none;
+  }
+
+  .header__line {
+	  display: none;
+  }
+
+  .main__title {
+    font-size: 90px;
+    margin-bottom: 10px;
+  }
+
+  .main__subtitle {
+    font-size: 15px;
+	    font-family: 'montserratmedium';
+    margin-bottom: 0;
+    padding: 0;
+    text-align: center;
+  }
+
+  .main__info {
+    display: flex;
+	    flex-direction: column;
+	    justify-content: center;
+	  align-items: center;
+  }
+
+  .main__content {
+	    margin: 0 0 40px 0;
+	    padding: 0;
+  }
+
+  .main__content-item {
+	    margin: 0 20px;
+  }
+
+  .content {
+    margin: 0;
+  }
+
+  .content__pic {
+	    width: 65px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)